### PR TITLE
Brand pivot

### PR DESCRIPTION
**Pull request by palmenhq**
Screenshot 2020-04-07 at 10 13 58
Screenshot 2020-04-07 at 10 14 07
Screenshot 2020-04-07 at 10 14 47
Screenshot 2020-04-07 at 10 15 07

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,9 +78,9 @@
     "@babel/plugin-proposal-class-properties": "^7.1.0",
     "@emotion/core": "^10.0.22",
     "@emotion/styled": "^10.0.23",
-    "@hedviginsurance/brand": "^3.0.5",
+    "@hedviginsurance/brand": "3.0.5",
     "@hedviginsurance/cli": "^2.3.2",
-    "@hedviginsurance/embark": "^1.0.66",
+    "@hedviginsurance/embark": "^2.0.0-alpha.3",
```
Comment (vonElfvin) [inline package.json:83]: We have to get 1.0.67 into this version before going live with combo

File: src/pages/OfferNew/index.tsx [hunk 2]
```diff
@@ -66,15 +86,24 @@ export const OfferNew: React.FC = () => {
...
+    )
   }
 
   const offerData = getOfferData(data?.quoteBundle! as QuoteBundle)
 
   return (
     <Page>
       <SessionTokenGuard>
-        <TopBar />
+        {![Variation.IOS, Variation.ANDROID].includes(variation!) && <TopBar />}
```
Comment (vonElfvin) [inline src/pages/OfferNew/index.tsx:106]: Nice catch

**Review by vonElfvin — APPROVED** [review]
Clean mean coding machine 🤖

File: package.json (hunk 1 above)
Comment (palmenhq) [reply package.json:83]: I merged that with `alpha.3` actually so i think it should work ™

**Pull request merged**